multiaccounts/settings_social_links/database.go

```diff
@@ -0,0 +1,84 @@
+package sociallinkssettings
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/status-im/status-go/protocol/identity"
+)
+
+type SocialLinksSettings struct {
+	db *sql.DB
+}
+
+func NewSocialLinksSettings(db *sql.DB) *SocialLinksSettings {
+	return &SocialLinksSettings{
+		db: db,
+	}
+}
+
+func (s *SocialLinksSettings) GetSocialLinks() (identity.SocialLinks, error) {
+	rows, err := s.db.Query(`SELECT link_text, link_url FROM social_links_settings`)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	links := identity.SocialLinks{}
+
+	for rows.Next() {
+		link := identity.SocialLink{}
+		var url sql.NullString
+
+		err := rows.Scan(
+			&link.Text, &url,
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		if url.Valid {
+			link.URL = url.String
+		}
+		links = append(links, link)
+	}
+
+	return links, nil
+}
+
+// Be careful, it removes every row except static links (__twitter, etc.) before insertion
+func (s *SocialLinksSettings) SetSocialLinks(links *identity.SocialLinks) (err error) {
```

Comment:
Consider doing a nil verification, to avoid someone mistakenly calling this with a nil object and having the code fail in the `for` loop in line 68
```
if links == nil {
     return errors.New(".......")
}
```